packages/dialect-react-ui/components/ChatButton/index.tsx

```diff
@@ -113,7 +112,7 @@ function WrappedChatButton(
           // className="dt-w-full dt-h-full bg-white/10"
           // style={{ backdropFilter: 'blur(132px)' }}
         >
-          <Chat />
+          <Chat toggleModal={() => setOpen((open) => !open)} />
```

Comment:
Question: why did you decide to go with "toggle" rather than usual "onClose" with this?

If I've checked everything correctly, we usually close it everywhere rather than "toggle" it. wdyt?

Reply:
kinda followed the similar function name on toggleSettings, but in this case I guess your suggestion to change to onClose is better fit. thanks, man!